src/EA.Weee.Web/Areas/Admin/Controllers/AatfController.cs

```diff
@@ -46,8 +49,18 @@ public async Task<ActionResult> Details(Guid id)
             {
                 AatfData aatf = await client.SendAsync(User.GetAccessToken(), new GetAatfById(id));
                 AatfContactData contactData = await client.SendAsync(User.GetAccessToken(), new GetAatfContact(id));
+                OrganisationData organisationData = await client.SendAsync(User.GetAccessToken(), new GetOrganisationInfo(aatf.Operator.OrganisationId));
 
```

Comment:
Is it possible to merge these API calls into a single API call, i.e. the GetAAtfById as that should be able to return the related entities, contact and address data. The the AatfData object can be expanded to have ContactData, Address Data, and List of related entities